libs/@local/hashql/syntax-jexpr/src/lexer/snapshots/hashql_syntax_jexpr__lexer__test__emoji_family.snap

```diff
@@ -0,0 +1,14 @@
+---
+source: libs/@local/hashql/syntax-jexpr/src/lexer/mod.rs
+description: Family emoji with zero-width joiners
+expression: 👨‍👩‍👧‍👦
```

Comment:
I appreciate you're testing multiple emojis, but isn't one enough? 😄

Reply:
Sadly not. Emoji are a mind field, and depending on the type of multi-byte emoji and how they're joined the representation in e.g. ariadne or during parsing is different. It's a world full of fun and adventure.